sycl/test/spec_const/spec_const_hw.cpp

```diff
@@ -5,7 +5,7 @@
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
 // TODO: re-enable after CI drivers are updated to newer which support spec
 // constants:
-// XFAIL: acc,cpu,cuda,gen
+// XFAIL: linux,cuda
```

Comment:
If I understand it correctly, today this test passes on host only. Right?
If so, I don't understand how did you manage to break it. :-)

Reply:
Or maybe I'm missing how new check works.
Does it say that it's failing on Linux **and** CUDA?

Reply:
I think it must be Linux **and** CUDA. It can't be Linux **or** CUDA, because it passes on the host.
So I suggest replace it with:
```suggestion
// UNSUPPORTED: cuda
```
And remove TODO comment above.

Please, apply to sycl/test/spec_const/spec_const_redefine.cpp as well.